tests/unit/test_i18n.py

```diff
@@ -25,7 +27,7 @@ def test_detect_locale(monkeypatch: MonkeyPatch, lang: str | None, expected: i18
         monkeypatch.setenv("LANG", lang)
 
     # Call the function and compare the result to the expected output.
-    result = i18n._detect_locale()  # pylint: disable=protected-access
+    result = i18n._detect_locale()
```

Comment:
can you please revert that change. I'd prefer to disable pylint rules locally and not globally for a fi.e